testdata/p4_16_samples_outputs/invalid-hdr-warnings4-frontend.p4

```diff
@@ -39,51 +39,29 @@ control IngressI(inout H hdr, inout M meta, inout standard_metadata_t smeta) {
     @name("IngressI.h") H h_0;
     @name("IngressI.h_copy") H h_copy_0;
     @name("IngressI.h_copy2") H h_copy2_0;
-    @name("IngressI.i") bit<1> i_0;
-    @name("IngressI.tmp") bit<1> tmp;
     @name("IngressI.hd") Header hd_0;
     @name("IngressI.hd") Header hd_1;
     @name("IngressI.hd") Header hd_8;
     @name("IngressI.hd") Header hd_9;
     @name("IngressI.hd") Header hd_10;
     @name("IngressI.hd") Header hd_11;
     @name("IngressI.hd") Header hd_12;
-    @name("IngressI.stack") Header[2] stack_0;
-    @name("IngressI.stack") Header[2] stack_2;
     @name("IngressI.validateHeader") action validateHeader() {
-        hd_10 = h_0.h1[0];
         hd_10.setValid();
-        h_0.h1[0] = hd_10;
     }
     @name("IngressI.validateHeader") action validateHeader_1() {
-        hd_11 = h_0.h2[tmp];
         hd_11.setValid();
-        h_0.h2[tmp] = hd_11;
     }
     @name("IngressI.invalidateHeader") action invalidateHeader() {
-        hd_12 = h_0.h2[0];
         hd_12.setInvalid();
-        h_0.h2[0] = hd_12;
```

Comment:
Some of these seem quite aggressive. Gauntlet does not complain but I will try to check these manually.